src/main/preload/modalPreload.js

```diff
@@ -75,6 +77,9 @@ window.addEventListener('message', async (event) => {
     case GET_LANGUAGE_INFORMATION:
         window.postMessage({type: RETRIEVED_LANGUAGE_INFORMATION, data: await ipcRenderer.invoke(GET_LANGUAGE_INFORMATION)});
         break;
+    case URL_OPEN_EXTERNAL:
```

Comment:
I don't think doing a message pass here makes a lot of sense since we can handle links the same way we do the Mattermost views. I've added a PR that will allow us to just have a regular link in the modal that should behave the same way that the Mattermost view links do: https://github.com/mattermost/desktop/pull/2247

Reply:
Updated to use regular link based on https://github.com/mattermost/desktop/pull/2247